### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by burmanm**
Update to linsk to k8ssandra repositories and k8ssandra docs instead of old Datastax resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-[![Go Report Card](https://goreportcard.com/badge/github.com/datastax/cass-operator)](https://goreportcard.com/report/github.com/datastax/cass-operator)
-[![License: Apache License 2.0](https://img.shields.io/github/license/datastax/cass-operator)](https://github.com/datastax/cass-operator/blob/master/LICENSE.txt)
+[![License: Apache License 2.0](https://img.shields.io/github/license/k8ssandra/cass-operator)](https://github.com/k8ssandra/cass-operator/blob/master/LICENSE.txt)
 
-The DataStax Kubernetes Operator for Apache Cassandra&reg;
+The k8ssandra Kubernetes Operator for Apache Cassandra&reg;. This repository replaces the old [datastax/cass-operator](https://github.com/datastax/cass-operator) for use-cases in the k8ssandra project. Some documentation is still out of date and will be modified in the future. Check [k8ssandra/k8ssandra](https://github.com/k8ssandra/k8ssandra) for more up to date information.
 
 ## Getting Started
 
+To create a full featured cluster, the recommend approach is to use the Helm charts. Check the [Getting started](https://k8ssandra.io/docs/getting-started/) documentation at (k8ssandra.io)[https://k8ssandra.io/docs].
```
Comment (bradfordcp) [inline README.md]: Should this go to a dedicated page covering how to just install the `cass-operator` helm chart?

For context, @johnsmartco has a section of the new docs site which will be dedicated to cass-operator.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # Cass Operator
-[![Gitter](https://badges.gitter.im/cass-operator/community.svg)](https://gitter.im/cass-operator/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-[![Go Report Card](https://goreportcard.com/badge/github.com/datastax/cass-operator)](https://goreportcard.com/report/github.com/datastax/cass-operator)
-[![License: Apache License 2.0](https://img.shields.io/github/license/datastax/cass-operator)](https://github.com/datastax/cass-operator/blob/master/LICENSE.txt)
+[![License: Apache License 2.0](https://img.shields.io/github/license/k8ssandra/cass-operator)](https://github.com/k8ssandra/cass-operator/blob/master/LICENSE.txt)
 
-The DataStax Kubernetes Operator for Apache Cassandra&reg;
+The k8ssandra Kubernetes Operator for Apache Cassandra&reg;. This repository replaces the old [datastax/cass-operator](https://github.com/datastax/cass-operator) for use-cases in the k8ssandra project. Some documentation is still out of date and will be modified in the future. Check [k8ssandra/k8ssandra](https://github.com/k8ssandra/k8ssandra) for more up to date information.
```
Comment (bradfordcp) [inline README.md]:
```suggestion
The K8ssandra Kubernetes Operator for Apache Cassandra&reg;. This repository replaces the old [datastax/cass-operator](https://github.com/datastax/cass-operator) for use-cases in the k8ssandra project. Some documentation is still out of date and will be modified in the future. Check [k8ssandra/k8ssandra](https://github.com/k8ssandra/k8ssandra) for more up to date information.
```

File: README.md [hunk 3]
```diff
@@ -154,64 +154,9 @@ cluster1-superuser@cqlsh> select * from system.peers;
...
-kubectl create namespace cass-operator-system
-helm install --set clusterWideInstall=true --namespace=cass-operator-system cass-operator ./charts/cass-operator-chart
-```
-
-#### Using a custom Docker registry with the Helm Chart
-
-A custom Docker registry may be used as the source of the operator Docker image.  Before "helm install" is run, a Secret of type "docker-registry" should be created with the proper credentials.
-
-Then the "imagePullSecret" helm value may be set to the name of the ImagePullSecret to cause the custom Docker registry to be used.
+To install a cluster with optional integrated backup/restore and repair utilities, check the [k8ssandra/k8ssandra](https://github.com/k8ssandra/k8ssandra) helm charts project.
```
Comment (bradfordcp) [inline README.md:158]: We should have a sentence referencing the full installation of K8ssandra, but still provide guidance on how to install just the `cass-operator` chart.

Not related to this PR, but interesting nonetheless, do you think there should be a Helm chart that simply templates out a `CassandraDatacenter`?

**Review by bradfordcp — COMMENTED** [review]
Left a few comments that are not directly blockers, but it would be nice to get some feedback prior to tossing a :+1: out there.

File: README.md (hunk 3 above)
Comment (jsanda) [reply README.md:158]: I do not think that there should be a standalone chart to template out a CassandraDatacenter. By doing that you effectively constraint yourself to applying updates only through Helm. If you update the dc outside of Helm and then later run `helm upgrade` your changes are lost.

**Review by johnsmartco — COMMENTED** [review]
@burmanm, thanks for the updates. Other than the product naming question on L4 of README.md, the edits in this PR look ok to me.

**Pull request merged**